vscode-dotty/package.json

```diff
@@ -14,7 +14,7 @@
     "vscode": "^1.27.1"
   },
   "categories": [
-    "Languages"
+    "Programming Languages"
```

Comment:
Why this change?

Reply:
Linting in vscode was suggesting that change, I guess it goes with their extensions category standards.